certora/harness/MorphoHarness.sol

```diff
@@ -22,6 +22,14 @@ contract MorphoHarness is Morpho {
         return idToMarketParams[id];
     }
 
+    function market_(Id id) external view returns (Market memory) {
+        return market[id];
+    }
+
+    function position_(Id id, address borrower) external view returns (Position memory) {
```

Comment:
```suggestion
    function position_(Id id, address user) external view returns (Position memory) {
```
as in `IMorpho`, because it can also be a supplier